vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -219,32 +168,41 @@ void CostMap::set_cost_map(const RoutingCosts& costs) {
             } else {
                 offset_[seg][box] = std::make_pair(seg_box_bounds.xmin(), seg_box_bounds.ymin());
                 cost_map_[seg][box] = vtr::NdMatrix<util::Cost_Entry, 2>(
-                    {size_t(seg_box_bounds.width()), size_t(seg_box_bounds.height())});
+                    {size_t(seg_box_bounds.width()), size_t(seg_box_bounds.height())}, util::Cost_Entry());
```

Comment:
This change shouldn't be required?  I believe the default argument is T(), e.g. util::Cost_Entry()?

Reply:
Fixed.